packages/rocketchat-integrations/server/api/api.coffee

```diff
@@ -167,7 +167,18 @@ executeIntegrationRest = ->
 				username: @user.username
 
 		try
-			result = script.process_incoming_request({ request: request })
+			sandbox =
+				_: _
+				s: s
+				console: console
+				Store:
+					set: (key, val) ->
+						return store[key] = val
+					get: (key) ->
+						return store[key]
+				script: script
+				request: request
+			result = vm.runInNewContext('script.process_incoming_webhook({ request: request })', sandbox, { timeout: 3000 })
```

Comment:
Why did you change the method name from `process_incoming_request` to `process_incoming_webhook`?